posthog/api/survey.py

```diff
@@ -993,20 +1022,22 @@ def survey_stats(self, request: request.Request, **kwargs) -> Response:
         Args:
             date_from: Optional ISO timestamp for start date (e.g. 2024-01-01T00:00:00Z)
             date_to: Optional ISO timestamp for end date (e.g. 2024-01-31T23:59:59Z)
+            ignore_cache: If True, ignore the cache and get fresh data from ClickHouse.
```

Comment:
do we have the same concept of ignoring cache in other APIs? or do the APIs either always cache or always return the real time data?

Reply:
I actually thought this through a little bit more, and the most expensive part of this endpoint, the query, is already cached in the query layer. So I decided to remove this cache and also this parameter, since there's no need to cache this thing twice